src/node/templates/default/template.ts

```diff
@@ -320,18 +310,18 @@ export const defaultTemplate: PkgTemplate = async ({cwd, logger, packagePath}) =
 
         pkgJson.devDependencies = {
           ...pkgJson.devDependencies,
-          'eslint': '*',
-          'eslint-config-prettier': features['prettier'] ? '*' : undefined,
-          'eslint-plugin-simple-import-sort': '*',
-          'eslint-plugin-import': '*',
-          'eslint-plugin-prettier': features['prettier'] ? '*' : undefined,
+          'eslint': '^8',
```

Comment:
ESLint v9 is not compatible with a lot of our plugins and configs yet. This is why the other semver ranges are also changed to specify the major version, to avoid this happening again in the future